src/pages/workspaces/workspace/Dashboard/InfoRow.ts

```diff
@@ -0,0 +1,18 @@
+import { PropsWithChildren, ReactNode } from 'react';
+import { dd, div, dt } from 'react-hyperscript-helpers';
+
+interface InfoRowProps extends PropsWithChildren {
+  title: string;
+  subtitle?: string;
+}
```

Comment:
It might be easier/clearer to write the type for `children` instead of using `PropsWithChildren`.

```suggestion
interface InfoRowProps {
  children?: ReactNode;
  title: string;
  subtitle?: string;
}
```